Example/GiniVision/ComponentAPIAnalysisViewController.swift

```diff
@@ -164,18 +164,18 @@ class ComponentAPIAnalysisViewController: UIViewController {
         guard errorButton.alpha != 1.0 else {
             return
         }
-        UIView.animateWithDuration(0.5) {
+        UIView.animate(withDuration: 0.5, animations: {
```

Comment:
Here we could also drop the `animations:` keyword and let the closure stand alone at the end. 
What style would you embrace? @nsobadjiev